apps/prairielearn/src/sync/syncFromDisk.ts

```diff
@@ -52,6 +58,15 @@ export async function checkSharingConfigurationValid(
   if (!config.checkSharingOnSync) {
     return true;
   }
+  const institutionId = await sqldb.queryRow(
+    sql.get_institution_id,
+    { course_id: courseId },
+    z.string(),
+  );
```

Comment:
We already have a `selectInstitutionForCourse` model function, let's use that instead.

Reply:
thanks, done in 7ed2a6ad1e